app/components/elements/forms/select_day_field_component.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Elements
+  module Forms
+    # Component for form day of the month fields
+    class SelectDayFieldComponent < Forms::FieldComponent
```

Comment:
We can drop the `Forms::` prefix on components in `app/components/elements/forms/`